controllers/chat/group/changeChannelDetail.js

```diff
@@ -2,6 +2,8 @@ const {StreamChat} = require('stream-chat');
 const {CHANNEL_TYPE_STRING} = require('../../../helpers/constants');
 const ErrorResponse = require('../../../utils/response/ErrorResponse');
 const {ResponseSuccess} = require('../../../utils/Responses');
+const UsersFunction = require('../../../databases/functions/users');
+const {User} = require('../../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `UsersFunction` and `User` are imported but it's not clear if they are used in a secure way. Make sure that the `findUserById` function properly sanitizes and validates its input to prevent SQL injection attacks or data leaks.
```diff
- const UsersFunction = require('../../../databases/functions/users');
- const {User} = require('../../../databases/models');
+ const UsersFunction = require('../../../databases/functions/users'); // Ensure this function is secure
+ const {User} = require('../../../databases/models'); // Ensure this model is used securely
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->